### PR TITLE
Adding changes to the SIG Release docs handbook as learned from 1.30 …

### PR DESCRIPTION
**Pull request by drewhagen**
adding changes to the SIG Release docs handbook as learned from 1.30 before the 1.31 cycle starts.

/kind cleanup
/kind documentation

### CODE REVIEW

File: release-team/role-handbooks/docs/Release-Timeline.md
Comment (drewhagen) [inline release-team/role-handbooks/docs/Release-Timeline.md:1]: Note to self: It looks like this needs a rebase

File: release-team/role-handbooks/docs/how-tos/Take-over-a-doc-pr.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Taking over a Doc PR
```
Comment (drewhagen) [inline release-team/role-handbooks/docs/how-tos/Take-over-a-doc-pr.md:1]: Hey @Princesso!
I know you requested some extra details on how I went about this. Let me know if this needs any additional clarifications. Thanks!

**Comment by katcosgrove** [comment]
Thanks for doing this, Drew! 

/approve

**Pull request merged**